en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/mutual-ssl-authentication.md

```diff
@@ -0,0 +1,132 @@
+- Configure the `mTLSAPIsEnabled` to `true` under the `[router.downstream.tls]` configuration to enable the mTLS APIs in the Choreo Connect.
+
+     ```
+      [router.downstream.tls]
+         trustedCertPath = "/etc/ssl/certs/ca-certificates.crt"  # Default path for client ca-certificates
+         mTLSAPIsEnabled = true
+     ```
+
+    !!! important
+        Make sure to **close** the **HTTP** port when using this feature, and leave only the **HTTPS** port open. This would prevent any attempts to bypass the mTLS authentication. You may close the port by following one of the steps given below.
+
+        <table>
+          <tr>
+            <th>Deployment</th>
+            <th>Configuration</th>
+          </tr>
+          <tr>
+            <td>Docker Compose</td>
+            <td>Set `router.listenerPort` to `0` in the `config.toml` file.</td>
+          </tr>
+          <tr>
+            <td>Kubernetes</td>
+            <td>Set `router.listenerPort` to `0` in the `config-toml-configmap` file.</td>
+          </tr>
+          <tr>
+            <td>Kubernetes with Helm Charts</td>
+            <td>
+            Remove the following section from the `templates/router-service.yaml` file.
+            ```yaml
+            - name: "http-router"
+              port: 9090
+              targetPort: 9090
+              protocol: TCP
+            ```
+            Or else, set `router.listenerPort` to `0` in a new `config-toml-configmap.yaml` file and use this file during deployment with the flag `--set-file wso2.deployment.adapter.configToml=<FILE_PATH_FOR_TEMPLATED_CONFIG_TOML>`. Make sure that the other values are also set correctly in the new Configuration file.
+            </td>
+          </tr>
+        </table>
+          
+
+{!includes/design/create-mtls-api.md!}
+
+!!! Important
+    To invoke mTLS enabled APIs that deployed in Choreo Connect, the CA certificates of the client's public certificates should be added as trusted certificates to the router.
+
+      - [Add a Certificate to Choreo Connect Router as a Trusted Certificate]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/security/tls/backend-certificates/#add-a-certificate-to-choreo-connect-router-as-a-trusted-certificate)
+
+    If you need to change the location of the volume mount, the `trustedCertPath` value under the `[router.downstream.tls]` in config.toml should also be changed.
+
+{!includes/design/invoke-mtls-api-using-postman.md!}
+
+### Invoke an API secured with Mutual SSL using cURL
+
+-   
+
+     ``` tab="Format"
+      curl -k --location -X GET "<API_URL>" -H  "accept: application/json" -H  "Authorization: Bearer <access-token>" --key <client_private_key> --cert <client_public_certificate>
+     ```
+
+     ``` tab="Example"
+      curl -k --location -X GET 'https://localhost:9095/test/1.0/foo' -H 'accept: applicaition/json' -H 'Authorization: Bearer  0ee9aa70-d631-3401-b152-521b431036ca' --key privateKey.key --cert example.pem
+     ```
+
+!!!Note
```

Comment:
The note syntax seems to be wrong. Please check

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7065/commits/4ac3b72b3278ab8265e5f4635499d5ea37db0c04